Product.Data.Access/Migrations/20221203121558_AddItemAndItemObjects.Designer.cs

```diff
@@ -0,0 +1,110 @@
+﻿// <auto-generated />
```

Comment:
Migrations should be in DbMigration Project where as you have put those in Access